compilers/concrete-optimizer/concrete-optimizer/src/optimization/dag/multi_parameters/partition_cut.rs

```diff
@@ -116,6 +116,7 @@ impl PartitionCut {
                     }
                     noise_origins[op_i] = origins;
                 }
+                #[allow(clippy::assigning_clones)]
```

Comment:
use "clone_from" ?

Reply:
As discussed, it does not work in practice:
```
error[E0502]: cannot borrow `noise_origins` as immutable because it is also borrowed as mutable
   --> concrete-optimizer/src/optimization/dag/multi_parameters/partition_cut.rs:120:53
    |
120 | ...   noise_origins[op_i].clone_from(&noise_origins[in...
    |       -------------       ----------  ^^^^^^^^^^^^^ immutable borrow occurs here
    |       |                   |
    |       |                   mutable borrow later used by call
    |       mutable borrow occurs here
```